pyswmm/subcatchments.py

```diff
@@ -574,6 +574,8 @@ def statistics(self):
         +---------------------+
         | Peak Runoff Rate    |
         +---------------------+
+        | Pollutant Buildup   |
```

Comment:
@katmratliff, so regarding these rST tables, would you mind doing me a favor?  I think we should list the actual dictionary keys (this is in Nodes, Links, System, and Subatchments modules).  If you don't want to do it, I can :-)  It's my fault afterall :-D

This table generator is really nice: 

https://www.tablesgenerator.com/text_tables

Reply:
@bemcdonnell hopefully I caught them all!